install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -1,66 +1,426 @@
+
+== Installing {product-title} on Red Hat Virtualization
+
+You can install {product-title} using the bastion virtual machine:
+
+. Log in to *_rhel-bastion_*.
+. Create *_vars.yaml_* with the following content and update its parameters:
++
+[source,yml]
+----
+---
+# For detailed documentation of variables, see
+# openshift_ovirt: https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_ovirt#role-variables
+# openshift installation: https://github.com/openshift/openshift-ansible/tree/master/inventory
+engine_url: https://<Manager_VQDN>/ovirt-engine/api
+engine_user: admin@internal
+engine_password: "{{ engine_password }}"
+engine_insecure: false
+engine_cafile: /etc/pki/ovirt-engine/ca.pem
+
+openshift_ovirt_vm_manifest:
+  - name: 'master'
+    count: 1
+    profile: 'master_vm'
+  - name: 'compute'
+    count: 0
+    profile: 'node_vm'
+  - name: 'lb'
+    count: 0
+    profile: 'node_vm'
+  - name: 'etcd'
+    count: 0
+    profile: 'node_vm'
+  - name: infra
+    count: 0
+    profile: node_vm
+
+# Currently, only all-in-one installation ("openshift_ovirt_all_in_one: true") is supported.
+# Multi-node installation (master and node VMs installed separately) will be supported in a future release.
+openshift_ovirt_all_in_one: true
+openshift_ovirt_cluster: Default
+openshift_ovirt_data_store: data
+openshift_ovirt_ssh_key: "{{ lookup('file', '/root/.ssh/id_rsa_ssh_ocp_admin.pub') }}"
+
+public_hosted_zone:
+# Uncomment to disable install-time checks, for smaller scale installations
+#openshift_disable_check: memory_availability,disk_availability,docker_image_availability
+
+qcow_url: <1>
+image_path: /var/tmp
+template_name: rhelguest7
+template_cluster: "{{ openshift_ovirt_cluster }}"
+template_memory: 4GiB
+template_cpu: 1
+template_disk_storage: "{{ openshift_ovirt_data_store }}"
+template_disk_size: 100GiB
+template_nics:
+  - name: nic1
+    profile_name: ovirtmgmt
+    interface: virtio
+
+debug_vm_create: false
+wait_for_ip: true
+vm_infra_wait_for_ip_retries: 30
+vm_infra_wait_for_ip_delay: 20
+
+node_item: &node_item
+  cluster: "{{ openshift_ovirt_cluster }}"
+  template: "{{ template_name }}"
+  memory: "8GiB"
+  cores: "2"
+  high_availability: true
+  disks:
+    - name: docker
+      size: 15GiB
+      interface: virtio
+      storage_domain: "{{ openshift_ovirt_data_store }}"
+    - name: openshift
+      size: 30GiB
+      interface: virtio
+      storage_domain: "{{ openshift_ovirt_data_store }}"
+  state: running
+  cloud_init:
+    root_password: "{{ root_password }}"
+    authorized_ssh_keys: "{{ openshift_ovirt_ssh_key }}"
+    custom_script: "{{ cloud_init_script_node | to_nice_yaml }}"
+
+openshift_ovirt_vm_profile:
+  master_vm:
+    <<: *node_item
+    memory: 16GiB
+    cores: "{{ vm_cores | default(4) }}"
+    disks:
+      - name: docker
+        size: 15GiB
+        interface: virtio
+        storage_domain: "{{ openshift_ovirt_data_store }}"
+      - name: openshift
```

Comment:
line 250-253 needs to be removed. The disk 'openshift' is probably not needed and also creates an ordering problem.